application/language/chinese_simplified/general_words_lang.php

```diff
@@ -117,8 +117,7 @@
 $lang['gen_hamradio_prefix'] = "Prefix";
 $lang['gen_hamradio_suffix'] = "Suffix";
 $lang['gen_hamradio_mode'] = '模式';
-$lang['gen_hamradio_dx'] = 'Dx';
-$lang['gen_hamradio_mode'] = 'Mode';
+$lang['gen_hamradio_dx'] = 'DX';
```

Comment:
Please check that. You removed "$lang['gen_hamradio_mode'] = 'Mode';"

Reply:
Did you pull the latest `dev` branch?

Reply:
Yes, line 119 and line 121 of the original file is duplicated with same key 'gen_hamradio_mode', this cause the string translation is overwritten.

Reply:
ahhh omg haha here in switzerland it's 8:42 am sorry my bad